index.html

```diff
@@ -245,6 +245,18 @@ <h2 class="contact-title">I'm always interested in hearing<span class="newline">
             <img src="assets/images/contact-frame3.svg" id="contact-frame3" alt="contact-frame3">
         </div>
     </section>
+    <section id="mobile-menu" class="d-none">
+        <div class="icon-wrapper">
+            <img src="assets/icons/close-menu-icon.svg" alt="close menu icon">
+        </div>
+        <div class="menu-list">
+            <ul>
+                <li><a href="#"> Portfolio </a></li>
+                <li><a href="#"> About </a></li>
+                <li><a href="#"> Contact </a></li>
+            </ul>
```

Comment:
- [ ] It is nice that you added the `<a>` tags to your menu options but it still does not redirects the user to the correct section of the page. Kindly add this functionality to your mobile menu and remember that when the user clicks on the menu options, the menu should also disappear.
 -
<img width="772" alt="image" src="https://user-images.githubusercontent.com/61919617/189692474-e7ea8f88-ff4f-4b40-aa10-b6847edce034.png">

Reply:
ok, got it, thanks